charts/rancher-backup-crd/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: rancher-backup-crd
+appVersion: 999
+version: 999
```

Comment:
hi @nickgerace, I am new to the development of a helm chart, please bear with me if my question is trivial: 

Any reason we bump the `apiVersion` from v1 to v2 here?  Is it just a placeholder?

Reply:
Not at problem :)
Without a `requirements.yaml`, I do not see a reason to keep `apiVersion` at `v1`. This chart is functional with Helm v3.
Reference for version bump: https://helm.sh/docs/faq/#chartyaml-apiversion-bump

Reply:
Thank you for the explanation! It makes sense to me.

Reply:
No problem. Actually you raise a good point that even though hand-testing proved fruitful, I want to look over this again and provide proof for this change.

Reply:
I looked it over again and performed a backup. It works as intended, but I will leave this "unresolved" for more reviewers to take a look.

Reply:
Confirmed, we are good to go.